examples/profile/tests/ProfileTest.php

```diff
@@ -54,6 +56,16 @@ public function shouldReturnUserProfile()
             new \DateTime('1980-01-01')
         );
 
+        $extraData = (new SandboxExtraDataBuilder())
+            ->withDataEntry(
+                (new SandboxAttributeIssuanceDetailsBuilder())
+                    ->withDefinition('some-definition')
+                    ->withExpiryDate(new \DateTime('2020-01-01T00:00:00Z'))
```

Comment:
Is it misleading to use an Expiry Date in the past? I guess the only way of avoiding that in future would be to have something like `DateTime.Now() + 1 day`.

Reply:
Fixed in 68b02be